bower.json

```diff
@@ -5,10 +5,16 @@
     "dist/rollbar.umd.js"
   ],
   "ignore": [
+    "dist/*.snippet*",
+    "dist/*.nojson*",
+    "dist/*.named-amd*",
```

Comment:
Why is `named-amd` removed? (https://github.com/rollbar/rollbar.js/pull/151)

Or have I misunderstood the context in which it's removed?